lib/src/connection.dart

```diff
@@ -214,7 +215,7 @@ class PostgreSQLConnection extends Object
   /// default query timeout will be used.
   Future transaction(
     Future Function(PostgreSQLExecutionContext connection) queryBlock, {
-    int commitTimeoutInSeconds,
+    int commitTimeoutInSeconds = 30,
```

Comment:
I'd rather keep these optional parameters as nullable and set a default value just before we use them.